runtime/vm/jvminit.c

```diff
@@ -6339,10 +6335,11 @@ isSSE2SupportedOnX86() {
 		OMRSIG_SIGACTION(SIGILL, NULL, &oldHandler);
 		OMRSIG_SIGNAL(SIGILL, (void (*)(int)) handleSIGILLForSSE);
 		osSupportsSSE = TRUE;
-		mxcsr = J9SSE2GetMXCSR();
+		U_32 mxcsr;
```

Comment:
We support some platforms that aren't using c99 compilers yet so locals need to be defined at the top of a scope.

Also, all locals should be given an initial value to protect against future refactoring that may read them before they're defined.

Reply:
I see. It has been updated.